### DISCUSSION TITLE
Small QoL improvement regarding storage

### DISCUSSION THREAD

**Discussion by SuggestionAccountHere** [FakeFishGames/Barotrauma#10972] [Ideas and suggestions]
**1)**  Make the one handed storage containers display their first (or last or manually chosen) item's 2D sprite so players can instantly have an idea of what's inside without hovering the mouse and reading the small item names. This would just let players speed up their storage hassle.

Although this may not really be efficient, as it could require many modified sprites for all the items (judging by the looks of inventory system).

And for that possibility I would suggest that you make it so that when hovered over, the storage containers replace their sprite with the aforementioned sample item's sprite.

**IF** that is also not efficient or possible, you could maybe let the storage containers display their item how the ammunition is shown inside an item.

**2)**  As others suggested the similar: Garbage Shute

**3)** Interact with nearest door keybind. (For when you are running around and keep hurdling towards doors.)

**Bonus:** upcoming creature name idea: "Shellmorph"

**Comment by MarkCringan** [comment]
A garbage shute might be problematic with griefers and traitor mode, as it would make disposing of items far too easy for those modes.

**Reply by SuggestionAccountHere** [reply]
Making it singleplayer only would remove griefers from the equation while still achieving its core purpose. The trash problem is mainly annoying on singleplayer. 

OR make it so only items with no deconstruction materials (like empty crates) can be put in the shute since I don’t think people would grief by disposing of useless material.

**Reply by Singularia** [reply]
It is already easy. All you need to do is to wait a moment of a wreck or a ruin salvaging mission (or even better, the abyss mining) and silently throw all the items you want out of the airlock by using a simple steel crate.
